solidity/dashboard/src/components/TBTCRewardsDataTable.jsx

```diff
@@ -1,84 +1,53 @@
-          <TokenAmount
-            currencyIcon={Icons.TBTC}
-            currencyIconProps={{ width: 15, height: 15 }}
-            amount={amount}
-            withMetricSuffix={false}
-            displayAmountFunction={(amount) => toTokenUnit(amount).toString()}
-            amountClassName="text-big text-grey-70 overflow-x-scroll"
-          />
-        )}
-      />
-      <Column
-        header="transaction hash"
-        field="transactionHash"
-        renderContent={({ transactionHash }) => (
-          <ViewInBlockExplorer type="tx" id={transactionHash} />
-        )}
-      />
-      <Column
-        header="deposit token id"
-        field="depositTokenId"
-        renderContent={({ depositTokenId }) => (
-          <AddressShortcut address={depositTokenId} />
-        )}
-      />
+    <DataTable data={data} itemFieldId="id" noDataMessage="No rewards history.">
+      <Column header="amount" field="amount" renderContent={AmountCell} />
+      <Column header="status" field="status" renderContent={renderStatus} />
+      <Column header="rewards period" field="rewardsPeriod" />
       <Column
         header="operator"
         field="operator"
-        renderContent={({ depositTokenId, operatorAddress }) => (
-          <OperatorCell
-            depositTokenId={depositTokenId}
-            operatorAddress={operatorAddress}
-            fetchOperatorByDepositId={fetchOperatorByDepositId}
-          />
-        )}
+        renderContent={renderOperator}
       />
     </DataTable>
   )
 }
 
-const OperatorCell = React.memo(
-  ({ depositTokenId, fetchOperatorByDepositId, operatorAddress }) => {
-    const [isFetching, setIsFetching] = useState(false)
-
-    const fetchOperatorByDeposit = useCallback(async () => {
-      setIsFetching(true)
-      await fetchOperatorByDepositId(depositTokenId)
-      setIsFetching(false)
-    }, [fetchOperatorByDepositId, depositTokenId])
-
-    if (operatorAddress) {
-      return <AddressShortcut address={operatorAddress} />
-    }
-
-    return (
-      <div className="flex row center">
-        <span className="text-fade-out">0x000</span>
-        <span
-          className="flex row center text-secondary cursor-pointer"
-          onClick={fetchOperatorByDeposit}
-        >
-          <Icons.Load style={{ marginRight: "0.5rem" }} />{" "}
-          {isFetching ? "Loading" : "Load address"}
-        </span>
+const AmountCell = ({ amount }) => {
+  return (
+    <div className="flex row center">
+      <Icons.KeepOutline
+        className="keep-outline--grey-40 mr-1"
+        width={32}
+        height={32}
+      />
+      <div>
+        <div>{displayAmount(amount)}&nbsp;KEEP</div>
+        <div className="flex row center">
+          <Icons.Rewards width={8} height={8} />
+          <span className="text-small text-grey-40">&nbsp;Reward</span>
+        </div>
       </div>
-    )
+    </div>
+  )
+}
+
+const renderStatus = ({ status }) => {
+  switch (status) {
+    case "AVAILABLE":
```

Comment:
Maybe we should use constants and put them in `constants.js` file instead of usign raw strings here?

Reply:
yeep 👍 we have already defined these statuses in `constants.js`. Will update!